client/http/api.go

```diff
@@ -73,6 +73,8 @@ const (
 	PProfGoroutine = "/pd/api/v1/debug/pprof/goroutine"
 	// Others
 	MinResolvedTSPrefix = "/pd/api/v1/min-resolved-ts"
+	ClusterPrefix       = "/pd/api/v1/cluster"
+	ClusterStatusPrefix = "/pd/api/v1/cluster/status"
```

Comment:
`XXXPrefix` here means there will be more content attached after this URI, it seems these two URIs are completed, so `Cluster` and `ClusterStatus` would be more suitable.

Reply:
get it